docs/chapters/remote/jinad.md

```diff
@@ -0,0 +1,279 @@
+           Flow@69676[S]:🎉 Flow is ready to use, accepting gRPC request
+           Flow@69676[I]:
+	🖥️ Local access:	tcp://0.0.0.0:51327
+	🔒 Private network:	tcp://192.168.1.9:51327
+	🌐 Public address:	tcp://203.184.132.69:51327
+         Client@69676[S]:connected to the gateway at 0.0.0.0:51327!
+...
+        gateway@69697[I]:send: 1 recv: 0 pending: 1
+pod0/ZEDRuntime@69694[I]:recv IndexRequest  from gateway▸pod0/ZEDRuntime▸⚐
+pod0/ZEDRuntime@69694[I]:#sent: 0 #recv: 1 sent_size: 0 Bytes recv_size: 423 Bytes
+🌏 pod1/ZEDRuntime@69696[I]:recv IndexRequest  from gateway▸pod0/ZEDRuntime▸pod1/ZEDRuntime▸⚐
+🌏 pod1/ZEDRuntime@69696[I]:#sent: 0 #recv: 1 sent_size: 0 Bytes recv_size: 508 Bytes
+...
+	✅ done in ⏱ 2 seconds 🐎 47.9/s
+        gateway@69676[S]:terminated
+...
+           pod1@69676[S]:terminated
+...
+           pod0@69676[S]:terminated
+...
+```
+
+### Creating a Remote Flow
+One common use case in production is running the whole Flow on the remote with pods distributed on different machines. In this case, the very first thing is to ensure all the remote machines have `jinad` running properly. We will see how to create a remote Flow from the local machine.
+
+#### 1. Create a Flow 
+As `jinad` hosts a service on the remote, we can use the `jinad` API `/flow/yaml` to create a Flow via uploading a Flow configuration yaml file. Please refere to the full  [API specifications](https://api.jina.ai/daemon/) for more details.
+
+```python
+import requests
+
+def create_flow(flow_url, yamlspec):
+    flow_creation_api = f'{flow_url}/flow/yaml/'
+    with open(yamlspec, 'rb') as f:
+        files = [('yamlspec', f)]
+        try:
+            r = requests.put(url=flow_creation_api, files=files, timeout=10)
+            if r.status_code == requests.codes.ok:
+                return r.json()['flow_id']
+            else:
+                print('Remote Flow creation failed')
+        except requests.exceptions.RequestException as ex:
+            print(f'Exception raised: {ex!r}')
+
+if __name__ == '__main__':
+    host_ip = '3.16.166.3'
+    jinad_port = '8000'
+    flow_api = f'http://{host_ip}:{jinad_port}'
+    flow_id = create_flow(flow_api, 'dummy_flow.yml')
+    print(f'flow is created: {flow_id}')
+```
+
+We use a dummy Flow configuration `dummy_flow.yml` as below for demo purpose.
+
+```yaml
+!Flow
+version: 1
+pods:
+  - name: pod0
+    method: add
+    uses: _logforward
+  - name: pod1
+    method: add
+    uses: _logforward
+```
+
+After running the above codes, we print out the flow id and please write down this flow id for the next step. 
+
+```text
+flow is created: cdd53e16-5575-11eb-86b2-0ab9db700358
+```
+
+#### 2. Check Flow StatusFlow 
+Before using the Flow we created, we need to get detail information about the Flow so that we can send queries to the Flow. To get the information of the Flow, we can use the following codes to retrieve the information from `/flow/cdd53e16-5575-11eb-86b2-0ab9db700358`
```

Comment:
I'm unsure of the usage of the word `codes` here. Would `script` be better?

Reply:
sounds better.